Atari-Game-Player/ReadMe.md

```diff
@@ -0,0 +1,45 @@
+# Atari - Solving Games with AI 🤖
+
+The purpose of this repository is to demonstrate how ZenML enables you to build complex applications in an easy way and Use Zenml for various classes of tasks.
+
+ZenML is an extensible, open-source MLOps framework to create production-ready machine learning pipelines. Built for data scientists, it has a simple, flexible syntax, is cloud- and tool-agnostic, and has interfaces/abstractions that are catered towards ML workflows.
+
+At its core, ZenML pipelines execute ML-specific workflows from sourcing data to splitting, preprocessing, training, all the way to the evaluation of results and even serving. There are many built-in batteries to support common ML development tasks. ZenML is not here to replace the great tools that solve these individual problems. Rather, it integrates natively with popular ML tooling and gives standard abstraction to write your workflows.
+
+Within this repo, we will use ZenML to build pipelines for developing One of the most powerful reinforcement learning algorithms which is DQN, which leans to solve Atari Games using AI.
+
+The demo for a trained model which solves Atari is following:-

```

Comment:
I'm not entirely sure what you mean here - does this mean that the gif shows a trained agent by this repo solving the game? - If yes we could formulate this sentence a bit differently to make it obvious.